### PR TITLE
fcaddr: reads a base16 encoded public key and prints the human readable address

### PR DESCRIPTION
**Pull request by travisperson**

### CODE REVIEW

File: cmd/fcaddr/main.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/filecoin-project/go-address"
+
+	"golang.org/x/xerrors"
+)
+
+const (
```
Comment (frrist) [inline cmd/fcaddr/main.go:16]: Not: consider defining theses in constant.go

**Review by frrist — APPROVED** [review]
Lgtm

File: cmd/fcaddr/main.go (hunk 1 above)
Comment (travisperson) [reply cmd/fcaddr/main.go:16]: I can put them in a constant.go file in this folder, but I wouldn't want to put them in the top level as they don't apply to the library itself but this small little application. They are strings here so they can be accepted as input on the CLI.

Comment (frrist) [reply cmd/fcaddr/main.go:16]: Sorry that was supposed to be a "Nit", I think what you've done here is fine, I was just being picky. :shipit:

**Review by magik6k — APPROVED** [review]

**Pull request merged**